backend/src/main/scala/cromwell/backend/standard/StandardCacheHitCopyingActor.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.backend.standard
+
+import java.nio.file.Path
+
+import akka.actor.ActorRef
+import cromwell.backend.callcaching.CacheHitDuplicating
+import cromwell.backend.{BackendCacheHitCopyingActor, BackendConfigurationDescriptor, BackendInitializationData, BackendJobDescriptor}
+
+/**
+  * Trait of parameters passed to a StandardCacheHitCopyingActor.
+  */
+trait StandardCacheHitCopyingActorParams {
+  val jobDescriptor: BackendJobDescriptor
+  val backendInitializationDataOption: Option[BackendInitializationData]
+  val serviceRegistryActor: ActorRef
+  val configurationDescriptor: BackendConfigurationDescriptor
+}
+
+/** A default implementation of the cache hit copying params. */
+case class DefaultStandardCacheHitCopyingActorParams
+(
```

Comment:
Yep, for our really long class names, intellij starts indentation two spaces after the `(`. Wrapping the `(` to the beginning of the line keeps the code from scrolling off the screen.